api-samples/spot-preemption-handling/README.md

```diff
@@ -0,0 +1,9 @@
+## Overview
+Batch tasks can be interrupted by Spot preemption. Batch now provides a new exit code (50001) for Spot preemption task failures. Below examples demonstarte how to make use of the new exit code to retry failed tasks.
+
+## Samples
```

Comment:
Can you add a submit.sh as other samples?

Reply:
added